pinot-segment-local/src/test/java/org/apache/pinot/segment/local/segment/index/NullValueIndexTest.java

```diff
@@ -37,13 +35,7 @@ public void oldToNewConfConversion() {
       _tableConfig.getIndexingConfig().setNullHandlingEnabled(true);
       convertToUpdatedFormat();
       assertNotNull(_tableConfig.getFieldConfigList());
```

Comment:
This shouldn't be a requirement of the test. Whether the fieldConfigList is null or not is not relevant for null vector type, given it does nothing.

What I would check is that getIndexingConfig().isNullHandlingEnabled is still true

Reply:
Makes sense, will add the check.